mlflow/protos/service.proto

```diff
@@ -75,6 +75,18 @@ service MlflowService {
     };
   }
 
+  rpc searchExperiments (SearchExperiments) returns (SearchExperiments.Response) {
+    option (rpc) = {
+      endpoints: [{
+        method: "POST",
```

Comment:
Let's see if we can support turning this into a `GET` endpoint by checking to see if that's compatible on Databricks. If so, we should change this to `GET`.

Reply:
@WeichenXu123 This is a new API, so we don't need to be concerned about backwards compatibility.